### PR TITLE
#87 (add remove instructor) into web-dev

### PR DESCRIPTION
**Pull request by dobber1611**
I put a gear on the nav bar for settings, but this could easily be changed to the users profile picture, which would be slick as well. 
image
image

### CODE REVIEW

**Comment by Awkewainze** [comment]
Build failed, otherwise looks fine, considering it's just mocked and not doing logic yet.

Also, quite a few angular warnings, have you look on how to clean those up?

**Comment by dobber1611** [comment]
@DylanWalseth I have looked a little into the angular warnings. I think that its having a problem with our JQuery as well (like materialize), because the angular.element is using the jqlite instead of our jquery, and then I get errors client side from jqlite.

https://docs.angularjs.org/api/ng/function/angular.element

The other warning is from adding a click event in the InstructorCtrl, and it uses this. Is there a different/better way to do this that you know of?

**Comment by Awkewainze** [comment]
http://www.w3schools.com/angular/ng_ng-click.asp
The example at the bottom would be closest to what you're trying to do.

Or maybe http://stackoverflow.com/a/21595931

It's my understanding that if you're using angular you really shouldn't need to directly call jquery ever. But I'm not an expert on it. Idk, experiment for a little bit and lmk what you find out.

**Comment by sorianog** [comment]
Build still failing but I like how the UI looks. Good work @dobber1611 !

File: web/public/js/controllers/InstructorsCtrl.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+			{ title: "Action",
+				defaultContent:
+					"<button class = 'waves-effect waves-light btn removeBtn' type=\"button\">Remove</button>",
+				width: "15%"}
+		]
+	});
+
+	$(".removeBtn").on("click", function (){
+		table.row($(this).parents("tr")).remove().draw();
+		// TODO Remove from DB
```
Comment (sorianog) [inline web/public/js/controllers/InstructorsCtrl.js]: Gotcha. I was about to ask if it does a remove from the db, but your comment says it is a TODO.

**Review by Awkewainze — APPROVED** [review]
gud nuff

**Pull request merged**